<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelGroupMember</class>
 <widget class="QWidget" name="SelGroupMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelGroupMember</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    font-family:微软雅黑;
}
QWidget#leftPanel{
    background-color:#EAEBED;
	border-bottom-left-radius: 4px;
	border-top-left-radius: 4px;
}
QWidget#rightPanel{
	border-bottom-right-radius: 4px;
	border-top-right-radius: 4px;	
    background-color:#FFFFFF;
}
QTreeWidget,QListWidget{
    border:none;
    background-color:#00000000;
}
QTreeWidget::branch{
	border-image:none;
	image:none;
}

QLineEdit#searchEdit{
    padding-left:20px;
    border:1px solid #FFFFFF;
    border-radius: 13px;
}

QLabel{
    color:#666666;
}

QPushButton#ok{
	 border-radius: 0px;
    color:#FFFFFF;
    background-color:#3DCE3D;
}
QPushButton#ok:disabled{
    background-color:#9DE69D;
}
QPushButton#ok:hover{
    background-color:#38C738;
}
QPushButton#ok:pressed{
    background-color:#2FB82F;
}

QPushButton#cancle{
    color:#666666;
    border:1px solid #E5E5E5;
}
QPushButton#cancle:hover{
    background-color:#f5f5f5;
}
QPushButton#cancle:pressed,#edit:pressed{
    background-color:#EBEBEB;
}

QPushButton#close{
	height:27px;
	width:30px;
	background:#00000000;
	border-image: url(:/Images/close_normal.png); 
}
QPushButton#close:hover{
	background:#F25561;
	border-image: url(:/Images/close_hover.png);  
}
QPushButton#close:pressed{
	background:#E74450;
	border-image: url(:/Images/close_hover.png);  
}



QPushButton#tabFriend,#tabDeps{
	height:27px;
	background:#00000000;
	border:none;
	color:#666666;
}

QPushButton#tabFriend:checked,#tabDeps:checked{
	border:none;
	color:#FFFFFF;
    background-color:#3DCE3D;
}
QWidget#widget,QWidget#widget_3
{
	border:none;
	color:#FFFFFF;
    background-color:#EAEBED;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="shadowbg" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" name="shadowinnerbg" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>550</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="opaqueResize">
            <bool>false</bool>
           </property>
           <property name="handleWidth">
            <number>0</number>
           </property>
           <property name="childrenCollapsible">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="leftPanel" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>261</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>261</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>48</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>48</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>16</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>16</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="searchEdit">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="placeholderText">
                   <string>搜索联系人</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="tabhodler" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="tabDeps">
                  <property name="text">
                   <string>群组成员</string>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>false</bool>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="memberpage">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="SlidingStackedWidget" name="memberstack">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="jiagoupage">
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QTreeWidget" name="jiagouTreeWidget">
                       <property name="minimumSize">
                        <size>
                         <width>10</width>
                         <height>10</height>
                        </size>
                       </property>
                       <column>
                        <property name="text">
                         <string notr="true">1</string>
                        </property>
                       </column>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="friendPage">
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QTreeWidget" name="friendList">
                       <column>
                        <property name="text">
                         <string notr="true">1</string>
                        </property>
                       </column>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="searchpage">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QTreeWidget" name="searchTreeWidget">
                   <column>
                    <property name="text">
                     <string notr="true">1</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="rightPanel" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_6" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>48</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>48</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>20</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="headcount">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>请勾选需要聊天的群成员:</string>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QWidget" name="widget_2" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="close">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="flat">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="listWidget"/>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>86</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>86</height>
                </size>
               </property>
               <widget class="QLabel" name="label">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>10</y>
                  <width>161</width>
                  <height>16</height>
                 </rect>
                </property>
                <property name="text">
                 <string>请输入音视频聊天室的名称:</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>40</y>
                  <width>341</width>
                  <height>31</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>20</number>
                </property>
                <property name="leftMargin">
                 <number>10</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>20</number>
                </property>
                <property name="bottomMargin">
                 <number>20</number>
                </property>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="cancle">
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>取消</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="ok">
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>确定</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
